test-cloud/pom.xml

```diff
@@ -27,6 +27,7 @@
     <openshift.admin.password>admin</openshift.admin.password>
 
     <kie.image.streams/> <!-- Needs to be defined for proper test run. -->
+    <cloud.properties.location/> <!-- Needs to be defined for proper test run. -->
```

Comment:
Actually, the idea is to add more properties in this file instead of adding more and more properties that require to modify the jenkins job. 

I will add in the README.md what properties are expected (by looking in the CloudProperties.java file indeed).